docs/developer-docs/latest/setup-deployment-guides/configurations.md

```diff
@@ -136,9 +138,6 @@ module.exports = ({ env }) => ({
           ca: fs.readFileSync(`${__dirname}/path/to/your/ca-certificate.crt`).toString(),
         },
       },
-      options: {
```

Comment:
The options key still needs to exist as an empty object. See: https://github.com/strapi/documentation/issues/290#issuecomment-842643273

Reply:
Resolved in last commit 👍🏼